differentiable_robot_model/rigid_body_params.py

```diff
@@ -38,38 +38,40 @@ def __init__(
         is_using_positive_initial_guest=False,
         init_param=None,
         is_initializing_params=True,
+        device='cpu'
```

Comment:
For `torch.nn.Modules`, it would be cleaner to call `to()` outside the module after initialization to set the device for all parameters within the module and its submodules, instead of having to specify the device at every layer.

Reply:
I had tried that initially but somehow it kept failing, I'll take a look again